docs/getting-started/installation.md

```
@@ -84,7 +86,35 @@ sudo systemctl enable cardwired --now
+sudo systemctl enable cardwired --now
+```
+
+## Ubuntu / Debian
+
+A `.deb` package is available on the [releases page](https://github.com/OpenGamingCollective/cardwire/releases). You can download and install it using `apt`.
+
+```bash
+sudo apt install ./cardwire_*.deb
+```
+
+Then [enable BPF LSM](#enabling-bpf-lsm) (if not already enabled by default) and start the service:
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)